packages/unit-testing/stubs/action/action.js

```diff
@@ -0,0 +1,127 @@
+     * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if options.recordType or options.id or options.params.recordId is missing or undefined
+     * @throws {SuiteScriptError} SSS_INVALID_RECORD_TYPE if the specified record type doesn't exist
+     * @throws {SuiteScriptError} SSS_INVALID_ACTION_ID if the specified action doesn't exist on the said record type
+     * @throws {SuiteScriptError} RECORD_DOES_NOT_EXIST if the specified record instance doesn't exist
+     *
+     * @since 2018.2
+     */
+    action.prototype.execute = function (options) { };
+    action.prototype.execute.promise = function (options) { };
+
+    /**
+     * Executes an asynchronous bulk record action and returns its task ID for status queries. The options.params parameter
+     * is mutually exclusive to options.condition and options.paramCallback.
+     *
+     * @governance 50 units
+     *
+     * @param {Object} options
+     * @param {string} options.recordType record type
+     * @param {string} options.id action ID
+     * @param {Array<Params>} options.params array of parameter objects; each object corresponds to one record ID for which the action is to
+     *                                             be executed; the object has the following form: {recordId: 1, someParam: 'foo', otherParam: 'bar'}
+     *                                             recordId is always mandatory, other parameters are optional and specific to the particular action
+     * @param {string=} options.condition condition used to select record IDs for which the action is to be executed; only the
+     *                                              action.ALL_QUALIFIED_INSTANCES constant is supported at the moment
+     * @param {string=} options.paramCallback function that takes record ID and returns the parameter object for the given record ID
+     * @return {string} task ID used in a later call to getBulkStatus
+     * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if options.recordType or options.id is missing or undefined
+     * @throws {SuiteScriptError} SSS_INVALID_RECORD_TYPE if the specified record type doesn't exist
+     * @throws {SuiteScriptError} SSS_INVALID_ACTION_ID if the specified action doesn't exist on the said record type
+     * @throws {SuiteScriptError} RECORD_DOES_NOT_EXIST if the specified record instance doesn't exist
+     *
+     * @since 2019.1
+     */
+    action.prototype.executeBulk = function (options) { };
+
+    /**
+     * Returns the current status of a bulk execution with the given task ID.
+     *
+     * @governance none
+     *
+     * @param {Object} options
+     * @param {string} options.taskId a task ID that was returned by a previous call to executeBulk
+     * @return {RecordActionTaskStatus} a status object capturing the current state of the bulk action execution; see task module JSDoc
+     *
+     * @since 2019.1
+     */
+    action.prototype.getBulkStatus = function (options) { };
```

Comment:
I don't see getBulkStatus method in N/action module but in action.Action (N/action/instance), I think we should remove it.

Reply:
It is in the N/action module according to the doc